plugins/modules/repository.py

```diff
@@ -27,6 +27,10 @@
 description:
   - Crate and manage repositories
 author: "Eric D Helms (@ehelms)"
+notes:
+  - You can configure certain aspects of existing Red Hat Repositories (like I(download_policy)) using this module,
+    but you can't create (enable) or delete (disable) them.
+  - If you want to enable or disable Red Hat Repositories, please use the M(theforeman.foreman.repository_set) module instead.
```

Comment:
```suggestion
  - If you want to enable or disable Red Hat Repositories available in your subscription manifest, please use the M(theforeman.foreman.repository_set) module instead.
```
Or should we even mention the `manifest` module here?

Reply:
I wouldn't mention the manifest here, no. I'd word it:

```
If you want to enable or disable Red Hat Repositories available through your subscription,
please use the M(theforeman.foreman.repository_set) module instead.
```